src/styles/components/_tag.scss

```diff
@@ -1,81 +1,76 @@
-@mixin tag-default
-{
+@mixin tag-default {
   display: block;
   @include font-family-base-heavy;
   @include font-size($fs-small);
+
   text-transform: uppercase;
   letter-spacing: 0.075rem; // NOTE: this is a one-off size
 }
 
-@mixin tag-assertive
-{
+@mixin tag-assertive {
   @include tag-default;
+
   display: inline;
   padding: space(md) space(lg);
-  -webkit-box-decoration-break: clone;
   box-decoration-break: clone;
 }
 
-@mixin tag-accent
-{
+@mixin tag-accent {
   @include tag-default;
+
+  border-width: 0 0 0 space(sm);
```

Comment:
i thought we had some weird cross-browser reason for specifying the individual properties. let's test this.